component/src/main/java/org/wso2/extension/siddhi/execution/geo/internal/impl/GeoCoordinateResolverHolder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.execution.geo.internal.impl;
+
+import org.wso2.extension.siddhi.execution.geo.api.GeoCoordinateResolver;
+
+/**
+ * A singleton class to initialize extension.
+ */
+public class GeoCoordinateResolverHolder {
+    private static GeoCoordinateResolverHolder geoCoordinateResolverHolder = new GeoCoordinateResolverHolder();
+    private GeoCoordinateResolver geoCoordinateResolver;
+
+    private GeoCoordinateResolverHolder() {
+
+    }
+
+    public static GeoCoordinateResolverHolder getGeoCoordinationResolverHolderInstance() {
+        return geoCoordinateResolverHolder;
+    }
+
+    public GeoCoordinateResolver getGeoCoordinateResolver(String geoResolverImplClassName)
+            throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+        if (geoCoordinateResolver == null) {
+            synchronized (this) {
+                geoCoordinateResolver = (GeoCoordinateResolver) Class.forName
```

Comment:
line 41 and 42 can be a single line

Reply:
Fixed https://github.com/wso2-extensions/siddhi-execution-geo/pull/61/commits/a08d5e724a57f053d7d18ae441ace09bafeb208a